app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,67 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: null,
+  tags: [],
 };
 
+export const ASSIGNED_CONTACT = [
+  {
+    label: "Admin",
+    value: "Admin",
+  },
+  {
+    label: "Non Admin",
+    value: "Non_Admin",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Geeting Started",
+    value: "Getting_Started",
+  },
+  {
+    label: "Meeting",
+    value: "Meeting",
+  },
+  {
+    label: "Learning",
+    value: "Learning",
+  },
+  {
+    label: "Onboarding",
+    value: "Onboarding",
+  },
+  {
+    label: "Leaves and Holidays",
+    value: "Leaves_and_Holidays",
+  },
+];
+
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
+  contact: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string().oneOf(ASSIGNED_CONTACT.map(contact => contact.label)),
+      value: yup.string().oneOf(ASSIGNED_CONTACT.map(contact => contact.value)),
+    })
+    .required("Assigned contact is required."),
```

Comment:
Keep the keyword as just `Contact`. `Assigned contact` is not an entity. He we are asking the user to assign a contact.